carsus/io/chianti_/chianti_.py

```diff
@@ -306,7 +320,8 @@ def __init__(self, session, ions=None, ds_short_name=None):
             else:
                 print("Ion {0} is not available".format(ion))
 
-        self.data_source = DataSource.as_unique(self.session, short_name=ds_short_name)
+        self.data_source = DataSource.as_unique(
+            self.session, short_name=ds_short_name)
```

Comment:
why is there a session - aren't we post DB

Reply:
Sorry, this change was introduced by autoPEP8